src/types/venues.ts

```diff
@@ -233,6 +233,10 @@ export interface PosterPageVenue extends BaseVenue {
   isLive?: boolean;
 }
 
+export interface PersonalizedPoster extends PosterPageVenue {
+  isSaved: boolean;
+}
+
```

Comment:
As mentioned in other comments, I believe this should also be reverted to how it was previously.

```suggestion
```

Reply:
Done